samples/sqltransport/store-and-forward/Version_6/Receiver/Program.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Data.SqlClient;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Features;
+using NServiceBus.Transports.SQLServer;
+#pragma warning disable 618
+
+class Program
+{
+    static void Main()
+    {
+        AsyncMain().GetAwaiter().GetResult();
+    }
+
+    static async Task AsyncMain()
+    {
+        Console.Title = "Samples.SqlServer.StoreAndForwardReceiver";
+        EndpointConfiguration configuration = new EndpointConfiguration("Samples.SqlServer.StoreAndForwardReceiver");
+
+        #region ReceiverConfiguration
+
+        configuration.UseTransport<SqlServerTransport>()
+            .EnableLagacyMultiInstanceMode(async transportAddress =>
+            {
+                string connectionString = transportAddress.StartsWith("Samples.SqlServer.StoreAndForwardReceiver") || transportAddress == "error"
```

Comment:
I am not sure what is the guidance now in the SQL Server transport. If I was using this feature, I would probably either use a convention or some shared mapping file.